.env

```diff
@@ -30,3 +30,4 @@ NEXT_PUBLIC_FOOTER_PRODUCT_LIST=http://localhost:3000/assets/products.json
 
 NEXT_PUBLIC_ONETRUST_PRIVACY_OTNOTICE_ID=d2b40527-50f9-4b2f-80ce-cf94247da0c0
 NEXT_PUBLIC_ONETRUST_COOKIES_CONSENT_OTNOTICE_ID=52635430-c9e6-4d7f-a265-73f7bd2d5dc0
+NEXT_PUBLIC_FF_WALLET_REVOCATION=false
```

Comment:
it looks like despite this feature flag is set to false users still see new UI elements: 
<img width="644" alt="image" src="https://github.com/pagopa/io-web-profile/assets/148224/44c914a9-a6e8-42da-9096-eec6ccdbd21c">